autogen

```diff
@@ -161,6 +162,13 @@ Options:
                            add a top-level documentation comment after the
                            license header (default: enabled)
 
+  --no-runline
+                           Skip adding runline code
```

Comment:
Move the docs one line higher; they're only one line lower when the flag is too long and would overrun this text, which it doesn't in this case.